repose-aggregator/components/filters/body-extractor-to-header/src/main/scala/org/openrepose/filters/bodyextractortoheader/BodyExtractorToHeaderFilter.scala

```diff
@@ -0,0 +1,121 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+package org.openrepose.filters.bodyextractortoheader
+
+import javax.servlet._
+import javax.servlet.http.HttpServletRequest
+
+import com.jayway.jsonpath.{DocumentContext, JsonPath, Configuration => JsonConfiguration, Option => JsonOption}
+import org.openrepose.commons.utils.servlet.http.MutableHttpServletRequest
+import org.openrepose.core.services.config.ConfigurationService
+import org.openrepose.filters.bodyextractortoheader.config.BodyExtractorToHeaderConfig
+
+import scala.collection.JavaConverters._
+import scala.io.Source
+import scala.util.{Failure, Success, Try}
+
+class BodyExtractorToHeaderFilter(configurationService: ConfigurationService)
+  extends AbstractConfiguredFilter[BodyExtractorToHeaderConfig](configurationService) {
+  override val DEFAULT_CONFIG: String = "body-extractor-to-header.cfg.xml"
+  override val SCHEMA_LOCATION: String = "/META-INF/schema/config/body-extractor-to-header.xsd"
```

Comment:
The Scala style guide recommends a different format for constants:
http://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods

Although, since these are inherited from the abstract filter, they might be legit. I guess the question is, can we use the abstract filter in Java? If so, then this is fine. If not, I'd adopt the Scala style.